libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
Which is the problem? The define is just a macro and at the level of the language we should replace it everytime there is a call and then parsing the code. If at the moment the parser does not allow it, I think we can leave the runtime error when it is encountered outside an install.